ts/src/ndax.ts

```diff
@@ -362,7 +362,12 @@ export default class ndax extends Exchange {
             const currency = response[i];
             const id = this.safeString (currency, 'ProductId');
             const name = this.safeString (currency, 'ProductFullName');
-            const type = this.safeString (currency, 'ProductType');
+            const ProductType = this.safeString (currency, 'ProductType');
+            let type = ProductType === 'NationalCurrency' ? 'fiat' : 'crypto';
```

Comment:
@ttodua can you add () to these conditions to avoid issues transpiling it?

Reply:
uh oh, forgot them. now done & pushed.